packages/tf-inventory/README.md

```diff
@@ -19,23 +19,28 @@ npm install @cumulus/tf-inventory
 The following utilities are provided via the CLI. For help run `tf-inventory --help`
 
 ```
-Usage: tf-inventory TYPE COMMAND [options]
-
-Options:
-  -V, --version            output the version number
-  -h, --help               output usage information
-
-Commands:
-  list-deployments         List Terraform Cumulus deployments in the account
-  deployment-report        List each Cumulus deployment with files, number of resources, and last update date
-  list-orphaned-resources  List resources not associated with a Terraform deployment, currently supports ECS and EC2
+  list-deployments [options]            List Terraform Cumulus deployments in the account
```

Comment:
Sorry, on second thought, I think the previous output might have been better, since it shows the usage more explicitly. Your call though